rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1693,7 +1702,7 @@ public String taskLog(@HeaderParam("sessionid") String sessionId, @PathParam("jo
     private String retrieveTaskLogsUsingDatabase(String sessionId, String jobId, String taskName)
             throws NotConnectedRestException, UnknownJobException, UnknownTaskException, NotConnectedException,
             PermissionException, PermissionRestException {
-        Scheduler scheduler = checkAccess(sessionId, "jobs/" + jobId + "/tasks/" + taskName + "/result/log/all");
+        Scheduler scheduler = checkAccess(sessionId, PATH_JOBS + jobId + PATH_TASKS + taskName + "/result/log/all");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "/result/log/all" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)